stable/handbrake/3.0.0/questions.yaml

```diff
@@ -150,6 +416,64 @@ questions:
                   type: string
                   default: "100Gi"
 
+        # Optical Drive
+        
+        - variable: opticaldrive
+          label: "Optical Drive Device"
+          schema:
```

Comment:
What the heck are you doing here? This is NOT persistence and will break if enabled.
You are trying to mix the old appVolumeMounts with Persistence.

You can NOT mount devices using the new persistence system. Please refer to things like zwave2mqtt or homeassistant on how to mount devices.

Reply:
> What the heck are you doing here? This is NOT persistence and will break if enabled.

It striked me while i was going to work in the morning :D I should sleep more at nights lol!